biosimulations/nx.json

```diff
@@ -131,11 +131,14 @@
     "shared-environments": {
       "tags": ["scope:shared", "type:environments", "platform:any"]
     },
-    "dispatch-datamodel": {
-      "tags": ["scope:dispatch", "type:datamodel", "platform:any"]
-    },
     "shared-datamodel-api": {
       "tags": ["scope:shared", "type:datamodel", "platform:any"]
+    },
+    "dispatch-api-models": {
+      "tags": ["scope:dispatch", "type:datamodel", "platform:any"]
+    },
+    "messages-dispatch": {
+      "tags": ["scope:shared", "type:messagequeues", "platform:any"]
```

Comment:
If the scope is intended to be shared. lets make this just messages. not message-dispatch. 
The type should be datamodel
The platform should be server

Reply:
done